business-registry-dashboard/app/components/Table/AffiliatedEntity/index.vue

```diff
@@ -5,6 +5,32 @@ const affStore = useAffiliationsStore()
 onMounted(async () => {
   await affStore.loadAffiliations()
 })
+
+// Watch for newly added businesses or name requests to highlight them temporarily
+watch(
+  () => affStore.newlyAddedIdentifier,
+  () => {
+    const firstResult = affStore.filteredResults?.[0] as any
+    const newIdentifier = affStore.newlyAddedIdentifier
+
+    // Skip if no results or no new identifier
+    if (!firstResult) { return }
+
+    // If the first result matches the newly added business or name request, highlight it
+    if (newIdentifier && (firstResult.businessIdentifier === newIdentifier || firstResult.nrNumber === newIdentifier)) {
+      firstResult.class = 'bg-[#E8F5E9]'
```

Comment:
Hex colour bad.

Reply:
Agreed. I'll be reaching out to see if Thor and Dietrich can let me to update the dependency and publish it. I'll get rid of all these hex colors.

Reply:
Thanks. You may resolve this conversation.